prompts.js

```diff
@@ -1,48 +0,0 @@
-const semver = require('semver');
```

Comment:
Why is this file removed? Just remove the prompts that are removed. Don't remove the whole file.

Reply:
Once the version number check has been removed (as we support v6 only) then the prompts file becomes redundant:

- Type only ever supported "init" - no need for a prompt if there's only one option
- semver check has been removed
- docs option is redundant as it comes as part of essentials
- csf is the default way to write stories, and you really shouldn't be turning it off in the options

I'd like to think that these should be a best-practice starting point, showing developers how things can be done. They're free to delete these template files when they don't need them. With 6.0, docs and csf on or off would have no effect on any dependencies being installed - just what templates are provided

Reply:
Yeah, but I want to do different types of generations. So, let's keep the type prompt

Reply:
ok